gateway-ha/src/main/java/com/lyft/data/gateway/ha/persistence/JdbcConnectionManager.java

```diff
@@ -21,11 +22,20 @@ public JdbcConnectionManager(DataStoreConfiguration configuration) {
   }
 
   public void open() {
+    this.open(null);
+  }
+
+  public void open(@Nullable String routingGroupDatabase) {
+    String jdbcUrl = configuration.getJdbcUrl();
```

Comment:
Why not add the schema to Use as a config to https://github.com/lyft/presto-gateway/blob/bc76b899a1fd75e885edbfdd508fdb488487381a/gateway-ha/src/main/java/com/lyft/data/gateway/ha/config/DataStoreConfiguration.java